webapp/channels/src/components/admin_console/server_logs/logs.tsx

```diff
@@ -41,13 +56,34 @@ export default class Logs extends React.PureComponent<Props, State> {
             logLevels: [],
             search: '',
             serverNames: [],
+            page: 0,
+            perPage: 1000,
+            loadingPlain: true,
         };
     }
 
     componentDidMount() {
-        this.reload();
+        if (this.props.isPlainLogs) {
+            this.reloadPlain();
+        } else {
+            this.reload();
+        }
     }
 
+    componentDidUpdate(prevProps: Props, prevState: State) {
+        if (this.state.page !== prevState.page) {
+            this.reloadPlain();
```

Comment:
Should we add a check for `this.props.isPlainLogs` here to make it explicit? I understand that `this.state.page` changes only when we're in plain logs but adding a check for `this.props.isPlainLogs` makes it explicit and clear for future self.